rest/taskana-rest-spring/src/test/java/pro/taskana/rest/WorkbasketDefinitionControllerIntTest.java

```diff
@@ -175,6 +234,8 @@ void testNoErrorWhenImportWithSameIdButDifferentKeyAndDomain() throws IOExceptio
     HttpEntity<MultiValueMap<String, Object>> requestEntity = new HttpEntity<>(body, headers);
     String serverUrl = restHelper.toUrl(Mapping.URL_WORKBASKETDEFIITIONS);
 
-    return template.postForEntity(serverUrl, requestEntity, Void.class);
+    ResponseEntity<Void> responseImport = template
+                                              .postForEntity(serverUrl, requestEntity, Void.class);
+    assertEquals(expectedStatus, responseImport.getStatusCode());
```

Comment:
this check was always done after the importRequest-function - so I added it internally.